hpobench/benchmarks/ml_mmfb/lr_benchmark.py

```diff
@@ -0,0 +1,127 @@
+from typing import Union, Tuple, Dict
+
+import ConfigSpace as CS
+import numpy as np
+from ConfigSpace.hyperparameters import Hyperparameter
+from sklearn.linear_model import SGDClassifier
+
+from hpobench.dependencies.ml.ml_benchmark_template import MLBenchmark
+
+
+class LRBaseBenchmark(MLBenchmark):
+    def __init__(self,
+                 task_id: int,
+                 rng: Union[np.random.RandomState, int, None] = None,
+                 valid_size: float = 0.33,
+                 data_path: Union[str, None] = None):
+
+        super(LRBaseBenchmark, self).__init__(task_id, rng, valid_size, data_path)
+        self.cache_size = 500  # TODO: Do we need this?
```

Comment:
I was recommended to have it since SVM can become super slow for larger datasets
is it causing any issues with containerization?

Reply:
No, but I just did not found where it is used. :-)

Reply:
https://github.com/automl/HPOBench/blob/PR_Multi-fidelity-tabular-benchmarks/hpobench/benchmarks/ml_mmfb/svm_benchmark.py#L70

Reply:
Jep. but this is the lr - benchmark. And it does not inherit from the svm benchmark.

Reply:
oh then I totally got it wrong
`cache_size` is apparently applicable only for SVM